libnavui-base/api/current.txt

```diff
@@ -408,6 +408,118 @@ package com.mapbox.navigation.ui.base.model.maneuver {
 
 }
 
+package com.mapbox.navigation.ui.base.model.route {
+
+  public final class RouteConstants {
```

Comment:
Are we sure that we want to expose _all_ of these? Exposing just the layer IDs would be fine to resolve this particular issue. Easier to expose more later if necessary, harder to hide.